### PR TITLE
Install postgres13

### PR DESCRIPTION
**Pull request by phamtriduy**
As the title says. Stop installing postgres-devel-9 and install postgres13 from amazon-linux-extras. Tested locally.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,9 +29,11 @@ RUN curl -sL https://rpm.nodesource.com/setup_"${NODEJS_VERSION}".x | bash - &&
       zlib-devel-1.2.* && \
     yum clean all && \
     rm -rf /var/cache/yum && \
+    # postgres
+    amazon-linux-extras install -y postgresql13
     # rbenv
     git clone https://github.com/rbenv/rbenv.git ~/.rbenv && \
-    git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+    git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build \
```
Comment (markgllin) [inline Dockerfile]: dont need the `\` here

Comment (phamtriduy) [reply Dockerfile]: Lol I noticed that, already removed

**Review by markgllin — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -30,6 +29,8 @@ RUN curl -sL https://rpm.nodesource.com/setup_"${NODEJS_VERSION}".x | bash - &&
       zlib-devel-1.2.* && \
     yum clean all && \
     rm -rf /var/cache/yum && \
+    # postgres
+    amazon-linux-extras install -y postgresql13
```
Comment (davegallant) [inline Dockerfile]:
```suggestion
    amazon-linux-extras install -y postgresql13 && \
```

```console
❯ hadolint Dockerfile
Dockerfile:35:5 unexpected 'g' expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input
```

Comment (phamtriduy) [reply Dockerfile]: Lol good catch

**Pull request merged**